python/feathub/processors/flink/flink_processor.py

```diff
@@ -93,14 +98,12 @@ def __init__(
         :param registry: An entity registry.
         """
         super().__init__()
-        self.config = config
+        self.config = FlinkProcessorConfig(config)
```

Comment:
Would it be better to use different variable names for the input parameter and the self.config? Maybe use `props` for `Dict` and use `config` for subclasses of `BaseConfig`?

Same for other constructors (e.g. FeathubClient) that accept `Dict` as the parameter.